lib/engine/game/g_1849/game.rb

```diff
@@ -595,7 +595,7 @@ class Game < Game::Base
 
         AFG_HEXES = %w[C1 H8 M9 M11 B14].freeze
         PORT_HEXES = %w[a12 A5 L14 N8].freeze
-        SMS_HEXES = %w[B14 C1 C5 H12 J6 M9 M13].freeze
+        SMS_HEXES = %w[B14 C1 C5 E1 H12 J6 M9 M13].freeze
```

Comment:
I don't understand why this hex was added. I think the only time this could possibly be used by the SMS is in some convoluted scenario involving the advanced rules for token purchasing, with some corporation closures involved.

Reply:
The issue that prompted that change was #4143. It seems legit to me, but I'm fairly new to 1849.

Reply:
Oh, yeah. Makes sense now.